.circleci/config.yml

```diff
@@ -119,12 +119,16 @@ commands:
           name: "Update app manifest"
           command: |
             sed -i  "s/instances: [0-9]\+/instances\: 1/" deploy_manifest.yml
+      - run:
+          name: "Find PR number"
+          command: |
+            PR_NUMBER=$(echo $CIRCLE_PULL_REQUEST | sed 's|https://github.com/||' | cut -d '/' -f 4)
+            echo "export PR_NUMBER=${PR_NUMBER}" > $BASH_ENV
```

Comment:
This feels a bit weird to me :tada:

Reply:
Yep - definitely - since found you can modify parameters if you define them on a step but since this is well tested I'll stick with this